### PR TITLE
fix(vite): correct iframe blocking

### LINKED ISSUE #79: fix(vite): support iframe blocking
### Environment

- Operating System: `Darwin`
- Node Version:     `v16.19.0`
- Nuxt Version:     `3.2.2`
- Nitro Version:    `2.2.2`
- Package Manager:  `yarn@1.22.17`
- Builder:          `vite`
- User Config:      `app`, `css`, `vite`, `modules`, `sanity`, `nitro`, `hooks`
- Runtime Modules:  `@nuxtjs/sanity@1.6.0`, `@pinia/nuxt@0.4.6`, `@vueuse/nuxt@9.12.0`, `@nuxtjs/device@3.0.0`, `@dargmuesli/nuxt-cookie-control@5.2.2`
- Build Modules:    `-`

### Reproduction

`nuxt.config.ts` settings this option `isIframeBlocked: true`

### Describe the bug

Setting `isIframeBlocked: true` does not block iFrames (e.g. Vimeo Player)

### Additional context

_No response_

### Logs

_No response_

### PR DESCRIPTION
**Pull request by darioferderber**
### 📚 Description

Resolves #79 

### 📝 Checklist

- [x] All commits follow the Conventional Commit format
- [x] The PR's title follows the Conventional Commit format

### CODE REVIEW

**Comment by dargmuesli** [comment]
I will review this later this week :)

File: src/replace.ts
Comment (dargmuesli) [inline src/replace.ts:1]: Could we extract this file as a separate vite plugin? Does it make sense to be used for other projects?

Comment (darioferderber) [reply src/replace.ts:1]: Sure we can, but there is already plugin named [vite-plugin-replace](https://www.npmjs.com/package/vite-plugin-replace) which it's missing [```enforce: 'pre'```](https://github.com/nuxt/nuxt/issues/20132#issuecomment-1500332302).
I'm not able to find a repo for vite-plugin-replace to write them an issue or create a PR, so I rewrote it to fit our needs.

Comment (dargmuesli) [reply src/replace.ts:1]: It must be this: https://github.com/leanupjs/vite-plugin-replace

Comment (darioferderber) [reply src/replace.ts:1]: Yup, that's it!
Last commit was 2 years ago, I'm not sure should I create a PR there or we can continue using it like this, what's your opinion?

Comment (dargmuesli) [reply src/replace.ts:1]: Always try a PR, if there is no response in 3 days tag deleonio and if there is none in 7 days (in total) we can use the current state for now!

Comment (darioferderber) [reply src/replace.ts:1]: [Here ](https://github.com/leanupjs/vite-plugin-replace/pull/2) it is, now we'll wait 🙂

**Comment by dargmuesli** [comment]
I've edited your PR description to say "Resolves" which automatically closes the mentioned issue once this PR is merged :raised_hands:

File: src/replace.ts
Comment (dargmuesli) [reply src/replace.ts:1]: As the linked PR receives no feedback, I'll resolve this thread, having added a notice on this to the file's head.

**Comment by dargmuesli** [comment]
I rewrote the commit messages to follow the conventional commit format for the automated release to create a proper changelog entry. Thank you for raising this PR! :raised_hands:

**Pull request merged**